### PR TITLE
magnum: Update fedora atomic image used by tempest

### PR DESCRIPTION
**Pull request by sayalilunkad**

### CODE REVIEW

File: scripts/lib/mkcloud-onhost.sh [hunk 1]
```diff
@@ -148,7 +148,7 @@ function onhost_cacheclouddata
                 echo "images/$a/other/magnum-service-image.qcow2"
                 echo "images/$a/other/manila-service-image.qcow2"
                 # these are the real ones, the above are just symlinks
-                echo "images/$a/other/fedora-23-atomic-7.qcow2"
+                echo "images/$a/other/Fedora-Atomic-26.x86_64.qcow2"
```
Comment (toabctl) [inline scripts/lib/mkcloud-onhost.sh]: Isn't `$a` already the architecture? then please remove it from the image name.

Comment (sayalilunkad) [reply scripts/lib/mkcloud-onhost.sh]: That's something @dirkmueller will need to change first if needed.

Comment (rsalevsky) [reply scripts/lib/mkcloud-onhost.sh]: Changed the img name.

**Comment by rsalevsky** [comment]
What about Cloud 7 testing? Do we need a special case here?

**Pull request merged**